File: Math, Logical Implememt/GCD, LCM/acmicpc.net/2609/Main.py
# import sys
# input = sys.stdin.readline
file = open("./01 ing/acmicpc.net/sample.txt", "r")
input = file.readline
A, B = list(map(int, input().split(" ")))

a, b = A, B
while b != 0:
    r = a % b
    a = b
    b = r

print(a)
g = a
l = g*(A/g) * (B/g)
print(int(l))

File: Math, Logical Implememt/GCD, LCM/acmicpc.net/2609/Main.js
"use strict";

const inputFilePath = "/sample.txt";
const input = require("fs")
  .readFileSync(__dirname + inputFilePath)
  .toString()
  .trim()
  .split("\n");
// const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");

const [A, B] = input[0].split(" ").map((x) => Number(x));
let [a, b] = [A, B];

while (b != 0) {
  const r = a % b;
  a = b;
  b = r;
}
const g = a;
console.log(a);
const l = g * (A / g) * (B / g);
console.log(l);
